lib/v2/datasource/remote/api/signup_repository.dart

```diff
@@ -35,13 +41,15 @@ class SignupRepository extends NetworkRepository {
   }
 
   Future<Result> unpackDynamicLink(String scannedLink) async {
-    final PendingDynamicLinkData? unpackedLink =
-        await FirebaseDynamicLinks.instance.getDynamicLink(Uri.parse(scannedLink));
+    final PendingDynamicLinkData? unpackedLink = await FirebaseDynamicLinks
+        .instance
+        .getDynamicLink(Uri.parse(scannedLink));
 
     if (unpackedLink == null) {
       return mapHttpError('Link is invalid');
     } else {
-      final Map<String, String> queryParams = Uri.splitQueryString(unpackedLink.link.toString());
+      final Map<String, String> queryParams =
```

Comment:
is this needed?

Reply:
I used VS Code and it didn't have the 120 length set up. It's fixed now.